ui/src/app/layout/navigation/navigation.component.html

```diff
@@ -1,74 +1,85 @@
-<app-navigation-search [force]="!isCollapsed" [dataflowAppRoles]="['ROLE_VIEW']"></app-navigation-search>
+<div *ngIf="featureInfo$ | async as featureInfo; else loading">
+  <app-navigation-search [force]="!isCollapsed" [dataflowAppRoles]="['ROLE_VIEW']"></app-navigation-search>
 
-<div class="sidebar-user" *ngIf="securityInfo.isAuthenticationEnabled" [dataflowAppRoles]="['ROLE_VIEW']">
-  <div class="sidebar-item" routerLinkActive="active" *ngIf="!securityInfo.isAuthenticated">
-    <a routerLink="login">
-      <i class="sidebar-fa fa fa-sign-in"></i>
-      <span class="text">Login</span>
-    </a>
+  <div class="sidebar-user" *ngIf="securityInfo.isAuthenticationEnabled" [dataflowAppRoles]="['ROLE_VIEW']">
+    <div class="sidebar-item" routerLinkActive="active" *ngIf="!securityInfo.isAuthenticated">
+      <a routerLink="login">
```

Comment:
I don't think we need that anymore. As soon as you try to access the UI and you are NOT authenticated, you will be redirected to the UAA server.